applications/CoSimulationApplication/custom_data_structure/pyKratos/variables.py

```diff
@@ -0,0 +1,32 @@
+from __future__ import print_function, absolute_import, division 
```

Comment:
i would have a helper_utility to generate this file automatically from kratos variables.

i mean, not to write it by hand...

Reply:
@RiccardoRossi  you mean from `variables.h` ?

Reply:
yes, you can obtain the list of double and vector variables from kratos. see e.g

https://github.com/KratosMultiphysics/Kratos/blob/master/kratos/python_scripts/print_vars_to_xml.py

Reply:
> yes, you can obtain the list of double and vector variables from kratos. see e.g
> 
> https://github.com/KratosMultiphysics/Kratos/blob/master/kratos/python_scripts/print_vars_to_xml.py

That file needs some cleanup xD